src/SDL_log.c

```diff
@@ -39,6 +39,11 @@
 #include <android/log.h>
 #endif
 
+#if defined(__NGAGE__)
+#undef va_copy
+#define va_copy(dst, src)   dst = src
```

Comment:
We recently moved the definition of va_copy to SDL_internal.h, you could add the ngage version there.